.github/workflows/ci.yml

```diff
@@ -32,8 +39,66 @@ jobs:
     - name: Run verification
       run: make check
 
-    - name: Build Docker image
-      run: make docker
+    - name: Get current timestamp
+      id: timestamp
+      run: echo "::set-output name=timestamp::$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
+
+    - name: Build APIClarity
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        platforms: linux/amd64,linux/arm64,linux/arm/v7
+        tags: ghcr.io/apiclarity/apiclarity:${{ github.sha }}
```

Comment:
@kaechele I don't think we should push on every PR. We can use the 'push false' option.